### PR TITLE
Inherited fields

### PR DESCRIPTION
**Pull request by markknol**
* fields and methods are ordered by inherited type
* added a specific function in the api, to end up with simple template markup
* to use `map.get` in the template we *need* `dce=no` in run.hxml
* the type in printFieldSignature is needed for correct links of inherited members
* all is invisible when there is no inheritance

image

### CODE REVIEW

File: themes/default/resources/index.js [hunk 1]
```diff
@@ -126,6 +126,15 @@ $(document).ready(function(){
 			$(this.parentElement).addClass("active");
 		}
 	});
+  
+	$("a.expand-button").click(function (e) {
+    var container = $(this).parent().next();
```
Comment (Simn) [inline themes/default/resources/index.js]: indent is off

File: run.hxml [hunk 2]
```diff
@@ -5,4 +5,4 @@
 -cp src
 -neko run.n
 -main dox.Dox
--dce full
+-dce no
```
Comment (Simn) [inline run.hxml]: Given that run.n is essentially a template host this change makes sense anyway.

**Comment by Simn** [comment]
This is nice! There's something wrong with the indentation but I'll accept it anyway because I'm such a rebel!

**Comment by markknol** [comment]
Thanks! Ill correct indentation later. 

I was just thinking, with the same data I can create a breadcrumb in the top of the page where it could say something like 

"extends MovieClip / Sprite / DisplayObject / EventDispatcher"

**Pull request merged**